src/modules/server.ts

```diff
@@ -23,8 +22,8 @@ class WebServer {
 
   private async mongoSetup (config = { useNewUrlParser: true, useUnifiedTopology: true }): Promise<void> {
     try {
-      await mongoose.connect(this.mongoUrl, config)
-      console.log('Successfully connected to mongodb')
+      await mongoose.connect('mongodb://' + `${process.env.MONGO_HOST}` + `${process.env.MONGO_PORT}` + '/local', config)
```

Comment:
How about creating a global variable forgo mongo_host and other for mongo_port ?